internal/host/hostcommands/stop_installation_cmd_test.go

```diff
@@ -5,7 +5,7 @@ import (
 
 	"github.com/go-openapi/strfmt"
 	"github.com/google/uuid"
-	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/v2"
```

Comment:
next time we should split this PR to two parts: upgrading to ginkgo v2 and changing the DB teardown. Otherwise it's very hard to review this

Reply:
The teardown change are because of v2

Reply:
so those changes are co-dependent?

Reply:
V2 removes the need to call Finish on the mock manager since go mock hooks into the ginkgo automatically

Without the need to call finish all the aftereach calls become rather useless as the only thing they do is delete the database which is also useless, so I've removed them

Technically it can be done separately

Also we can do the podman / db init refractor separately

Reply:
I prefer doing those kind of "useless" changes, wherever it can make reviewing the changes easier
In general it's almost never useless updating APIs/versions, because it has additional benefits and features that we can leverage
Either way, I do think this PR is doing great stuff and improve a lot. It's just the process of review that I'm commenting on (and you'd be having less/easier git-conflicts that way :smiley: )